lib/server.coffee

```diff
@@ -169,6 +167,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 							# Extract JSON stringified phantomJS response after removing other potential Phantom logging messages. This regex extracts just the JSON.
 							try
 								output = JSON.parse stdout.replace /^(?!(\{.*\})$)(.*)|\r\n/gim, ''
```

Comment:
I found out that in some situations the stdout is empty. This happens mostly when we have tracking iframes created by external js (e.g. ads, intercom, ...). One way I could handle this, was to check in the isReady for the content to not be empty as well.